core/control-plane/contract/src/main/java/org/eclipse/dataspaceconnector/contract/offer/ContractOfferServiceImpl.java

```diff
@@ -53,12 +54,14 @@ public ContractOfferServiceImpl(ParticipantAgentService agentService, ContractDe
 
     @Override
     @NotNull
-    public Stream<ContractOffer> queryContractOffers(ContractOfferQuery query, Range range) {
+    public Stream<ContractOffer> queryContractOffers(ContractOfferQuery query) {
         var agent = agentService.createFor(query.getClaimToken());
 
-        return definitionService.definitionsFor(agent, range)
+        return definitionService.definitionsFor(agent, query.getDefinitionsRange())
                 .flatMap(definition -> {
-                    var assets = assetIndex.queryAssets(definition.getSelectorExpression());
+                    var assetFilterQuery = QuerySpec.Builder.newInstance()
```

Comment:
minor nit: static imports, i.e. `concat()` instead of `Stream.concat()`

Reply:
Done.